src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -12,7 +12,6 @@
  * The UI component that is responsible for receiving user command inputs.
  */
 public class CommandBox extends UiPart<Region> {
```

Comment:
Perhaps you want to do the logic of autocomplete here within the commandbox rather than in the MainWindow? Or have CommandBox have a AutoCompleteWordHandler or something since the commandTextField is in this class?